### DISCUSSION TITLE
Setup working nginx configuration

### DISCUSSION THREAD

**Discussion by laclaro** [jcorporation/myMPD#334] [Q&A] [answered]
Hello,

this is a support, rather than a development question. I tried to setup mympd behind nginx using the info from #93.

I could not make it run.

My configuration is as follows
```
upstream mympd {
  server    localhost:8082;
  keepalive 512;
}
...
location /mympd/ {
    proxy_pass http://mympd;
    proxy_set_header Accept-Encoding "";
    sub_filter_types text/html text/css text/javascript text/xml application/x-javascript;
    sub_filter_once off;
    sub_filter '/css/' '/mympd/css/';
    sub_filter '/js/' '/mympd/js/';
    sub_filter '/assets' '/mympd/assets';
    sub_filter '/api' '/mympd/api';
    sub_filter '/ws' '/mympd/ws';
    sub_filter '/player.html' '/mympd/player.html';
    sub_filter 'href="/' 'href="/mympd/';
}
```

It results in
```
myMPD error

Embedded asset /mympd/css not found
```

Could we setup a small working configuration in the wiki?

Best,

Henning

**Comment by jcorporation** [comment, marked as answer]
It is already documented: https://github.com/jcorporation/myMPD/wiki/Accessing-myMPD-through-Nginx

You don‘t need the sub_filter directives.

**Comment by laclaro** [comment]
I feel so dumb. I saw it and tried it and thought it did not work. However, now I did it again, and voilà. Thank you.
